Makefile

```diff
@@ -41,6 +43,9 @@ $(TSC):
 bower_components:
 	bower install
 
+lint:
+	$(TSLINT) $(foreach source,$(TSSOURCES), -f $(source))
```

Comment:
tslint accepts multiple files at a time, so we can simplify this to `$(TSLINT) -f $(TSSOURCES)`.